src/main/java/no/sikt/nva/scrapers/EntityDescriptionExtractor.java

```diff
@@ -59,6 +60,15 @@ public static EntityDescription extractEntityDescription(DublinCore dublinCore)
         return entityDescription;
     }
 
+    public static List<Contributor> extractContributors(DublinCore dublinCore) {
+        return dublinCore.getDcValues().stream()
+                   .filter(DcValue::isContributor)
+                   .map(EntityDescriptionExtractor::createContributorFromDcValue)
+                   .flatMap(Optional::stream)
+                   .filter(EntityDescriptionExtractor::hasName)
```

Comment:
I stedet for filter på slutten her: ta heller å oppdater createContributorFromDcValue til å returnere Optional.empty hvis contributor mangler navn.